perma_web/api/tests/test_link_resource.py

```diff
@@ -655,7 +655,7 @@ def test_should_create_archive_from_pdf_file(self):
                                        user=self.org_user)
 
             link = Link.objects.get(guid=obj['guid'])
-            self.assertRecordsInArchive(link, upload=True, filetype='warc')
+            self.assertRecordsInArchive(link, upload=True, filetype='wacz')
```

Comment:
With this change, I suspect that having `filetype` as a parameter of `assertRecordsInArchive` is now unnecessary, and that the filetype is now always wacz, and that whole branch can be removed. Do you agree?